docs/source/markdown/podman-network.1.md

```diff
@@ -35,5 +35,19 @@ so networks have to be created again after a backend change.
 | rm         | [podman-network-rm(1)](podman-network-rm.1.md)                 | Remove one or more networks                                     |
 | update     | [podman-network-update(1)](podman-network-update.1.md)         | Update an existing Podman network                               |
 
+## SUBNET NOTES
+Podman requires specific default IPs and, thus, network subnets.  The default values used by Podman can be modified in the **[containers.conf(5)](https://github.com/containers/common/blob/main/docs/containers.conf.5.md)** file.
+
+### Podman network
+The default bridge network (called `podman`) uses 10.88.0.0/16 as a subnet. When Podman runs as root, the `podman` network is used as default.  It is the same as adding the option `--network bridge` or `--network podman`. This subnet can be changed in **[containers.conf(5)](https://github.com/containers/common/blob/main/docs/containers.conf.5.md)** under the [network] section. Set the `default subnet` to any subnet that is free in the environment. The default `podman` name can also change to something else with the default network key. Note that this should only be done when no containers are running.
```

Comment:
The second to last sentence is a little wierd. "The name of the default network can also be changed from `podman` to another name using the default network key" maybe?

Reply:
Changed, with just a slight tweak to backtick the word podman.  That does flow better, thanks!